app/controllers/provider/admin/cms/templates_controller.rb

```diff
@@ -12,6 +12,8 @@ def new
 
   def edit
     @page ||= templates.find(params[:id])
+
+    render layout: false if request.headers['X-PJAX']
```

Comment:
See https://github.com/defunkt/jquery-pjax?tab=readme-ov-file#server-side-configuration

AFAIK only the edit page uses pjax, not new and not versions index.